e2e/cypress/integration/collapsed_reply_threads/crt_settings_spec.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+describe('CRT Settings', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
+        cy.apiInitSetup({loginAfter: true, promoteNewUserAsAdmin: true}).then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+        });
+    });
+
+    beforeEach(() => {
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+    });
+
+    it('should be able to toggle CRT on/off', () => {
```

Comment:
Please add exact Zephyr test case ID and title for all test cases. You may ask help from QA in-charge if you need help with Zephyr.